Source/WebCore/platform/graphics/filters/FELighting.h

```diff
@@ -62,6 +62,8 @@ class FELighting : public FilterEffect {
 protected:
     FELighting(Type, const Color& lightingColor, float surfaceScale, float diffuseConstant, float specularConstant, float specularExponent, float kernelUnitLengthX, float kernelUnitLengthY, Ref<LightSource>&&);
 
+    bool operator==(const FilterEffect&) const override { RELEASE_ASSERT_NOT_REACHED(); }
```

Comment:
Can you try this using statement instead:
```
    using FilterEffect::operator==;
```

Reply:
It works too, thanks!